lib/workers/pr/changelog/release-notes.ts

```diff
@@ -260,6 +264,31 @@ export async function getReleaseNotesMd(
   return null;
 }
 
+/**
+ * Determine how long to cache release notes based on when the version was released.
+ *
+ * It's not uncommon for release notes to be updated shortly after the release itself,
+ * so only cache for about an hour when the release is less than a week old. Otherwise,
+ * cache for days.
+ */
+export function releaseNotesCacheMinutes(releaseDate?: string | Date): number {
```

Comment:
should normally not a `Date`, but need type fixes, which i will send after this pr